lib/puppet/type/file/checksum.rb

```diff
@@ -33,8 +33,6 @@ def sum_stream(&block)
     "{#{type}}#{checksum}"
   end
 
-  private
-
   # Return the appropriate digest algorithm with fallbacks in case puppet defaults have not
   # been initialized.
   def digest_algorithm
```

Comment:
I was wrong about this, it has defaultto the setting. I was just testing it incorrectly. I'll restore `private`.